python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -676,12 +685,20 @@ export default function EditorContainer({
           <Tooltip
             label={isDirty ? "Save changes to config" : "No unsaved changes"}
           >
-            <div>
+            <div className={classes.toolbar}>
+              <Image
```

Comment:
Can you do this image/toolbar in the EditorContainer instead? That will solve 2 things:
- as is, this will have the AIConfig image show up in gradio when we use this component there
- should fix the grey bar issue

Reply:
Thanks, will amend a change.

Reply:
Done